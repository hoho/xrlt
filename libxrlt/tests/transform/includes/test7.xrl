<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">
    <xrl:response>
        <xrl:include>
            <xrl:href>/tmp</xrl:href>
            <xrl:type>text</xrl:type>
            <xrl:success test="/text() != 'hello'">hooray</xrl:success>
            <xrl:failure>oops</xrl:failure>
        </xrl:include>

        <xrl:variable name="ololo">
            <xrl:include>
                <xrl:href>/tmp2</xrl:href>
                <xrl:type>text</xrl:type>
                <xrl:success test="/text() = 'hello'">hooray</xrl:success>
                <xrl:failure>oops</xrl:failure>
            </xrl:include>
        </xrl:variable>

        <xrl:include>
            <xrl:href>/tmp3</xrl:href>
            <xrl:type>text</xrl:type>
            <xrl:success test="$ololo != 'oops'">hooray</xrl:success>
            <xrl:failure>oops</xrl:failure>
        </xrl:include>
    </xrl:response>
</xrl:requestsheet>
