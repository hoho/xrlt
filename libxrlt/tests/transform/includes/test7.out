XRLT_STATUS_SUBREQUEST
sr id: 1
sr method: GET
sr type: TEXT
sr url: /tmp
sr query: (null)
sr body: (null)
XRLT_STATUS_SUBREQUEST
sr id: 2
sr method: GET
sr type: TEXT
sr url: /tmp3
sr query: (null)
sr body: (null)
XRLT_STATUS_SUBREQUEST
sr id: 3
sr method: GET
sr type: TEXT
sr url: /tmp2
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: oops
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: hooray
XRLT_STATUS_DONE
